pkg/k8shandler/reconciler.go

```diff
@@ -68,14 +74,20 @@ func Reconcile(requestCluster *logging.ClusterLogging, requestClient client.Clie
 
 	// Reconcile Collection
 	if err = clusterLoggingRequest.CreateOrUpdateCollection(); err != nil {
+		clusterLoggingRequest.Cluster.Status.Conditions.SetCondition(condInvalid("Unable to create or update collection"))
 		return fmt.Errorf("Unable to create or update collection for %q: %v", clusterLoggingRequest.Cluster.Name, err)
```

Comment:
@alanconway this is the flip of logstore, while maybe not getting new logs, having deployed storage may still provide query into already collected logs

Reply:
+1 degraded

Reply:
ok